src/v2/view-builder/views/TerminalView.js

```diff
@@ -2,10 +2,31 @@ import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 import BaseFooter from '../internals/BaseFooter';
-import {getBackToSignInLink} from '../utils/LinksUtil';
+import BaseHeader from '../internals/BaseHeader';
+import HeaderBeacon from '../components/HeaderBeacon';
+import { getBackToSignInLink } from '../utils/LinksUtil';
+import { getIconClassNameForBeacon } from '../utils/AuthenticatorUtil';
 
 const RETURN_LINK_EXPIRED_KEY = 'idx.return.link.expired';
 
+const EMAIL_AUTHENTICATOR_TERMINAL_KEYS = [
+  'idx.transferred.to.new.tab',
+  'idx.return.to.original.tab',
+  RETURN_LINK_EXPIRED_KEY,
+  'idx.return.stale',
+  'idx.return.error'
+];
```

Comment:
I agree that, per a discussion on Slack, it may be a good idea to add `idx.email.verification.required` to this list.

(Strictly speaking, that key / terminal state is produced by the Activation machinery, not by the Email Authenticator, but its goal is to verify the user's supposed Email address by sending them an Email message, etc etc, so my 0.02 is that it would make sense to use the Email beacon for it -- but @haishengwu-okta may be a better judge)